### PR TITLE
Announcing 1 new detection and toggling 1 detection

### PR DESCRIPTION
**Pull request by unknownhad**
Announcing 1 new detection and toggling 1 detection

### CODE REVIEW

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ layout: list
...
-      <td>2022-06-20</td>
       <td>2022-07-05</td>
+      <td>2022-07-11</td>
       <td>Cloudflare Specials</td>
-      <td>...34780914</td>
-      <td>100532</td>
-      <td>Vulnerabilty scanner activity</td>
+      <td>...N/A</td>
+      <td>100533</td>
+      <td>web shell activity</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>Web shell activity</td>
```
Not sure if it's possible to adjust the capitalization here, even if it doesn't match the capitalization in the rule description.

File: content/waf/change-log/2022-07-05.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <th>Previous Action</th>
+      <th>New Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Cloudflare Specials</td>
+      <td>...34780914</td>
+      <td>100532</td>
+      <td>Vulnerabilty scanner activity</td>
```
Comment (unknownhad) [inline content/waf/change-log/2022-07-05.md]:
```suggestion
      <td>Vulnerability scanner activity</td>
```

**Review by helloandre — APPROVED** [review]

File: content/waf/change-log/scheduled-changes.md [hunk 3]
```diff
@@ -24,14 +24,14 @@ layout: list
...
   <tbody>
     <tr>
-      <td>2022-06-20</td>
       <td>2022-07-05</td>
+      <td>2022-07-11</td>
       <td>Cloudflare Specials</td>
-      <td>...34780914</td>
-      <td>100532</td>
-      <td>Vulnerabilty scanner activity</td>
+      <td>...N/A</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]: Are we still missing an ID here? If not, I'd remove the `...`.

Comment (unknownhad) [reply content/waf/change-log/scheduled-changes.md]: We update the N/A with IP once we have it.
`...N/A`  is just a placeholder.

**Review by unknownhad — COMMENTED** [review]
Updating schedule changes file

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**